vijaynadimpalli/ps.py

```diff
@@ -0,0 +1,437 @@
+import numpy as np
+import pandas as pd
+import scipy
+import itertools
+from statsmodels.distributions.empirical_distribution import ECDF
+import time
+import seaborn as sns
+import matplotlib.pyplot as plt
+from utils_multiprocess import run_traditional_correlation_calcs, run_extended_correlation_calcs, \
+    run_diagonal_measure_calcs, run_extremal_measure_calcs
+from ps_utils import get_sum_correlations, multivariate_rho, diagonal_measure, extremal_measure, get_co_variance_matrix
+
+
+class PartnerSelection:
+    """
+    Implementation of the Partner Selection procedures proposed in Section 3.1.1 in the following paper.
```

Comment:
Awesome that you are matching our style of Docstrings! Well done on doing your homework.